src/test/java/io/cdap/plugin/http/sink/batch/HTTPSinkConfigTest.java

```diff
@@ -67,6 +75,32 @@ public void testInvalidUrl() {
     assertPropertyValidationFailed(failureCollector, HTTPSinkConfig.URL);
   }
 
+  @Test
+  public void testMacroUrl() throws Exception {
```

Comment:
This is not the approach on how we test with Macro. 

Please take this as a reference and change your test accordingly:

https://github.com/cdapio/hydrator-plugins/blob/29a16d6e9f70447144930be47b85a37c99c1cc87/core-plugins/src/test/java/io/cdap/plugin/batch/sink/TableSinkTest.java#L255

Reply:
PR updated.